src/main/java/shukaro/warptheory/handlers/warpevents/WarpBlink.java

```diff
@@ -61,13 +61,13 @@ public void onTick(TickEvent.WorldTickEvent e)
                     int targetY = (int)player.posY + e.world.rand.nextInt(16) - e.world.rand.nextInt(16);
                     int targetZ = (int)player.posZ + e.world.rand.nextInt(16) - e.world.rand.nextInt(16);
                     BlockCoord target = new BlockCoord(targetX, targetY, targetZ);
-                    if (target.isAir(e.world) && target.copy().offset(1).isAir(e.world) && !target.copy().offset(0).isAir(e.world))
+                    if (target.isAir(e.world) && target.copy().offset(1).isAir(e.world) && target.copy().offset(0).isTopSolid(e.world))
                     {
-                        player.rotationPitch = (e.world.rand.nextInt(90) + e.world.rand.nextFloat()) - (e.world.rand.nextInt(90) + e.world.rand.nextFloat());
```

Comment:
why does this needs to be changed?

Reply:
Specifically not required.
I took it this effect simulate madness driving victim to wander helplessly.
Halve the pitch random range because it's the real pain when "fighting" against the warp effect, and that large a pitch isn't a typical one used when walking a lot.
Alternatively thought about limited pitch delta, such as :

player.rotationPitch += 30.0 * (0.5 - e.world.rand.nextFloat()); // -15 .. 15

Fear not, the effect is still a pain. ;)